traffic_ops/app/lib/API/Job.pm

```diff
@@ -113,6 +113,29 @@ sub show {
 	$self->success( \@data );
 }
 
+
+sub delete {
```

Comment:
I think you need to check tenancy on this method. you can't delete a job outside of your tenancy, right? the job table has a job_deliveryservice column that you can use to check tenancy...

Reply:
Correct, 
it was also missing in the show command.
Need some further testing before pulling